src/game/scenes/action/ElevatorGameScene.ts

```diff
@@ -0,0 +1,230 @@
+import gameManager, { Emitter } from '../../manager/GameManager'
+import MinigameScene from '../MinigameScene'
+import { scenesKeys } from '../../../utils/constants'
+import gameStore from '../../../store/GameStore'
+import { GameEvents } from '../../../utils/enums'
+import { gameWait } from '../../../utils/functions'
+
+export default class ElevatorGameScene extends MinigameScene {
+  // Texture
+  private elevatorWinTexture: string
+  private floorsTexture: string
+  private playerTexture: string
+  private playerIrritatedTexture: string
+  private playerArmsTexture: string
+  private playerMusicNoteTexture: string
+  private playerWaterDropTexture: string
+  private caseCallElevatorTexture: string
+  // Sprite
+  private elevator?: Phaser.GameObjects.Sprite
+  private elevatorDoors?: Phaser.GameObjects.Sprite
+  private floorsElevator?: Phaser.GameObjects.Sprite
+  private elevatorContainer?: Phaser.GameObjects.Container
+  private player?: Phaser.GameObjects.Sprite
+  private playerArms?: Phaser.GameObjects.Sprite
+  private playerMusicNote?: Phaser.GameObjects.Sprite
+  private playerWaterDrop?: Phaser.GameObjects.Sprite
+  private caseCallElevator?: Phaser.GameObjects.Sprite
+  private playerContainer?: Phaser.GameObjects.Container
+  // Variable
+  private currentFrame: integer = 1
+  private hitCaseCallElevator: integer = 0
+  private nbrHitCaseCall: integer = 2
+
+  constructor() {
+    super({
+      key: scenesKeys.ElevatorGame,
+    })
+
+    this.elevatorWinTexture = 'elevatorWinAnimation'
+    this.floorsTexture = 'floorsAnimation'
+    this.playerTexture = 'tokiWhistleAnimation'
+    this.playerArmsTexture = 'tokiArmsAnimation'
+    this.playerMusicNoteTexture = 'tokiMusicNoteAnimation'
+    this.playerWaterDropTexture = 'tokiWaterDropAnimation'
+    this.caseCallElevatorTexture = 'caseCallElevatorAnimation'
+    this.playerIrritatedTexture = 'tokiIrritatedAnimation'
+  }
+
+  public onFailure(): void {
```

Comment:
Juste supprime les méthodes `onFailure` et `onSuccess` parce que maintenant y a déjà du code écrit dans `MinigameScene` qui s'occupe de faire perdre une vie si tu perds, et charger la bonne scène `PostMinigame` etc. Ce qu'on fait désormais c'est on peut encore overrides ces méthodes là si tu dois faire quelque chose de particulier, mais actuellement vu ce que tu fais (tu charges juste en gros le prochain minijeu), supprime les, le `MinigameScene` se chargera de faire ce qu'il faut 👍 

(et quand je dis supprime les, je parle juste des déclarations de méthodes, tu laisses bien sûr ton appel à `this.onSuccess()` pour notifier quand on considère que le joueur a gagné)

Reply:
C'est fait 👍